scripts/start.sh

```diff
@@ -82,20 +82,36 @@ fi
 # Argument passed to dev_appserver.py to indicate whether or not to
 # clear the datastore.
 CLEAR_DATASTORE_ARG="--clear_datastore=true"
+# Argument passed to gulpfile.js to help build with minification
+MINIFICATION=false
 for arg in "$@"; do
   if [ "$arg" == "--save_datastore" ]; then
     CLEAR_DATASTORE_ARG=""
   fi
+  # if you want to start run oppia in production enviroment
+  # while in development environment
+  if [ "$arg" == "--prod_env" ]; then
+    MINIFICATION=true
+    $PYTHON_CMD scripts/build.py
+  fi
 done
 
+yaml_env_variable="MINIFICATION: $MINIFICATION"
```

Comment:
Nope, GAE resets all environ variables and hence we are not able to get it in feconf.py